src/DotNetStatus/config.json

```diff
@@ -1,3 +1,6 @@
 {
-    "ApiPortService": "https://portability.cloudapp.net"
+    "ApiPortService" : {
+        "LiveSiteUrl": "https://portability.cloudapp.net",
```

Comment:
This should instead be "ApiPortService": { "Url" : "https://portability.clouadapp.net" } and then have a config.debug.json that gets conditionally loaded if it's running in debug that sets the "url" field to the debug endpoint.